sample_files/sample_CSV/test_grades.py

```diff
@@ -0,0 +1,61 @@
+# import requests
+# from requests.auth import HTTPBasicAuth
+from verify import parse_assigned_zones, get_actual_zone_dict
+
+actual_dict = get_actual_zone_dict()
+assigned_dict = parse_assigned_zones()
+
+output_list = []
+file_list = ['TestExam_1']
+for file in file_list:
+    with open('%s.csv' % (file, ), 'r') as zones:
+        # Get header row contents
+        header_str = zones.readline()
+
+        line_list = zones.readlines()
+
+        # line_list = line_list[:1]
+        for index, line in enumerate(line_list):
+            line = line.strip()
+            if len(line) > 0 and 'Missing' not in line:
+                temp_array = []
+                record = line.split(',')
+                student_rcs = record[1]
+                student_name = record[0]
+                student_tmp = student_name.rsplit(' ', 1)
```

Comment:
A last name can have multiple "words" in it, so don't use `rsplit(' ',1)` on real data if it needs to be accurate. There's not really a reliable way to figure out where to chop up a name if it's more than 2 parts, so it's fine for an automatic attempt to break up a name.